src/components/ThirdPartyCollectionDetailPage/CollectionContextMenu/CollectionContextMenu.tsx

```diff
@@ -24,7 +24,7 @@ export default class CollectionContextMenu extends React.PureComponent<Props> {
 
   handleEditURN = () => {
     const { collection, onOpenModal } = this.props
-    onOpenModal('EditCollectionURNModal', { collection })
+    if (!collection.isPublished) onOpenModal('EditCollectionURNModal', { collection })
```

Comment:
The `pointer-events: auto;` allows the user to click the `Dropdown.Item`, so we need to check it manually here before opening the modal. Same for the `EditItemURNModal` modal.

Reply:
👍 
I know I'm annoying, but to be consistent with the rest of the codebase, would you mind adding `{}` to the if here?